step.yml

```diff
@@ -286,6 +286,17 @@ inputs:
       value_options:
       - "yes"
       - "no"
+  - cache_level: swift_packages
+    opts:
+      category: Debug
```

Comment:
Why is it in the `Debug` category?

Reply:
Which category would be ok? Create a cache one?